stockAndPrices/services/app/main.py

```diff
@@ -85,13 +88,12 @@ def dblogin():
             doc = dev_col.insert_one({"first": first, "last": last})
             rtn = "A document is inserted with your inputs: '" + first + " " + last + \
                   " and its corresponding object _id is " + str(doc.inserted_id)
+            http_code = 200
```

Comment:
The same logic applies here. The http_code is set to 200 already, and does not need to be set again.